test/libsolidity/syntaxTests/immutable/getter.sol

```diff
@@ -1,5 +1,5 @@
 contract C {
-    uint immutable public x;
+    uint immutable public x = 1;
```

Comment:
Do we also have semantic tests for that? I think that still needs to be implemented.

Reply:
Can you create a new pull request just with a test that calls a getter for an immutable variable?

Reply:
Note that this test result is "Unimplemented" so this is indeed still missing. Do you want the semantic test PR anyway?

Reply:
yes, as a reminder that it still needs to be implemenetd.

Reply:
Can you move this test to a different PR and change the expectations so that it fails?